taglib/ape/apetag.cpp

```diff
@@ -65,13 +65,7 @@ namespace
         return false;
     }
 
-    const String upperKey = String(key).upper();
-    for(auto k : invalidKeys) {
-      if(upperKey == k)
-        return false;
-    }
-
-    return true;
+    return std::none_of(invalidKeys.begin(), invalidKeys.end(), [upperKey = String(key).upper()](auto k) { return upperKey == k; });
```

Comment:
Would `const auto &k` make more sense in the parameter list? Most of the other suggested lambdas use const references for objects so I wonder what is different about this one.

Reply:
auto vs const auto& boils down to whether or not the parameter is trivially copyable. No real difference except the former being shorter.